src/App.jsx

```diff
@@ -14,70 +15,144 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState(['Jam']);
```

Comment:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```
it can be the state with simple string value, instead of the array